### PR TITLE
New version: iMobieInc.Ashampoo version 26.00.22

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

* Resolves #131693

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132554)

### CODE REVIEW

**Comment by Trenly** [comment]
Must be something weird about the azure agents

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 1/3/2024 8:09:58 PM
 Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   CC65F50E4DC257A21BEAFD4CB7BEA1AEBF201A7037436812ABED61A8AA274368
 Expected hash: 0568524591c97eb3d38d84cd711d98d7cc0f8a2992ecaf6b15a7eb8b6faf8d4d

(Automated response - build 516.)

File: manifests/i/iMobieInc/Ashampoo/26.00.22/iMobieInc.Ashampoo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: iMobieInc.Ashampoo
+PackageVersion: 26.00.22
+InstallerType: inno
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cdn1.ashampoo.net/public/imo/6706/ashampoo_winoptimizer_26_38898.exe
+  # InstallerUrl: https://www.imobie.com/go/download.php?product=ashampoo&id=0
+  InstallerSha256: 0568524591C97EB3D38D84CD711D98D7CC0F8A2992ECAF6B15A7EB8B6FAF8D4D
```
Comment (stephengillie) [inline manifests/i/iMobieInc/Ashampoo/26.00.22/iMobieInc.Ashampoo.installer.yaml:11]:
```suggestion
  InstallerSha256: CC65F50E4DC257A21BEAFD4CB7BEA1AEBF201A7037436812ABED61A8AA274368
```

Comment (Trenly) [reply manifests/i/iMobieInc/Ashampoo/26.00.22/iMobieInc.Ashampoo.installer.yaml:11]: @stephengillie - Can you check the hash you get when you download it to your VM? I always get 8D4D

Comment (stephengillie) [reply manifests/i/iMobieInc/Ashampoo/26.00.22/iMobieInc.Ashampoo.installer.yaml:11]:
```PowerShell
PS C:\ManVal> get-fileHash '.\Installers\ashampoo_winoptimizer_26_38898 (1).exe'

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          0568524591C97EB3D38D84CD711D98D7CC0F8A2992ECAF6B15A7EB8B6FAF8D4D       C:\ManVal\
```

Comment (stephengillie) [reply manifests/i/iMobieInc/Ashampoo/26.00.22/iMobieInc.Ashampoo.installer.yaml:11]: Unfortunately, the hash has to match at the pipelines, or critical scans can't be performed.

**Comment by stephengillie** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/131693

**Pull request closed without merging**